### DISCUSSION TITLE
Welcome to ExpirationDateTracker Discussions!

### DISCUSSION THREAD

**Discussion by ishuonly** [ishuonly/ExpirationDateTracker#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Annie-1-code** [comment]
Hi, my name is Anindita Bhattacharya. I want to join this project because I am curious about machine learning. How do I acquaint myself with the code base?

**Reply by ishuonly** [reply]
Hey, glad to hear that!

So first familiarize yourself with the project's overall objective and different pages and layers by reading the readme of the main branch. Then for the ML part, we have used Image Processing, to explore that part download the code base from the receipt-scanner and fresh-produce branches, and you'll get a fair idea of the models we used. But I would highly recommend you research and try out different models that can be used to improve efficiency in reading the receipts and capturing fruits, vegetables, etc. (can get the data through Kaggle).

**Reply by Annie-1-code** [reply]
Hi, I read the README.md and READMe_frontend.md. How to download the code from receipt-canner and fresh-produce branches?

**Reply by ishuonly** [reply]
Just go their respective branches and download the code :)

**Reply by Annie-1-code** [reply]
I have downloaded the code from the 2 branches  mentioned above.Could you tell me what the code in receipt scane is doing?

**Reply by Annie-1-code** [reply]
Also how to run the full code and check which files are run one after the other?

**Comment by Vishal-Gori** [comment]
Hi, my name is Vishal Gori. I found the project's real-life use case interesting and would like to contribute to it. How can I get familiar with the code base?

**Reply by ishuonly** [reply]
Hey, check the above comment, and then go through [Contribution Guidelines](https://github.com/ishuonly/ExpirationDateTracker/blob/main/CONTRIBUTION.md).

Tip: Checkout different branches for different functionalities of the project.

**Comment by NiharJani2002** [comment]
Hiiii @ishuonly . This seems to be an interesting project.
Would you share the specific dataset used for building the model ? Along with the link.
Dataset will help to  understand data more.

**Reply by ishuonly** [reply]
Hey, you can try the following dataset: https://www.kaggle.com/datasets/moltean/fruits

Although you can try out different datasets as well, just the objective should be satisfied

**Comment by Rukkya** [comment]
Hii, I'm rukya and i find this project really interesting especially if it could be scaled to large scale of users and be a cross-platform app as well, I'm down to code but I wonder whether there's a discord server for the project or such? i guess GitHub discussion won't be the most suitable form of suggesting ideas and discussing implementation in detail. what do you think?

**Reply by ishuonly** [reply]
Hey! Definitely, there wasn't any server till now, but since there are many people interested in this project, I'll create a discord server and share the link.

**Comment by ishuonly** [comment]
Here's the link to the discord server: https://discord.gg/HaN9m65M

Let's continue our further discussions over there. See ya!

**Reply by Annie-1-code** [reply]
hey, I can't login to the discord seve. It says site can't be reached

**Comment by Aakash1312-cmd** [comment]
Hello!!!! My name is Akash Dutta , currently pursuing my BTech from NIT Durgapur in third semester.I am curious about Machine Learning and Data Science and want to contribute dedicatedly towards this particular project of KWOC'23.I just want to know that what should I do next in order to start contributing towards this project 'ExpirationDateTracker'.

Looking forward for your reply and guidance from your side!!!!


Also can you please send me the discord channel link so that we can start discussion about the project

Thank You
Akash Dutta

**Reply by ishuonly** [reply]
Hey, glad to hear that, go through the contribution guidelines, try running the code and then feel free to ask in case of any doubt.

**Reply by Aakash1312-cmd** [reply]
Where can I ask, let me know.If I need to ask in Discord, help me to join in the discord community as the link of the discord community is not working and showing invalid...please help me

**Reply by ishuonly** [reply]
https://discord.gg/eDAeaNKGXb

Here's the link that'll work!

**Comment by 17AtGitHub** [comment]
Hii, I am Daksh. 4th yr UG from IITKGP. I am willing to contribute to the application development part of the project using MERN, and learn integrations of the application with the ML model. I have done some self-projects earlier, but never worked in a team. As an exciting project idea, I am willing to work on this, and discuss how to build this together. Should I start with something specific ?
Thanks

**Reply by ishuonly** [reply]
Hey Daksh! I'll create a few issues soon; please check them out. And if you have any more ideas, feel free to ping:)

**Comment by ishuonly** [comment]
The new [discord link!](https://discord.gg/eDAeaNKGXb)

**Comment by Aakash1312-cmd** [comment]
Can u please send the discord link?

**Reply by ishuonly** [reply]
Hey, check above!

**Comment by ishuonly** [comment]
@Everyone

Checkout the issues tab, I've raised a few issues. Lemme know if someone wants to take them up, I'll assign them to you. For further discussions, join the discord server. See you there ;)

**Reply by Aakash1312-cmd** [reply]
okay
